source/includes/usage-examples/sample-app-intro.rst

```diff
@@ -0,0 +1,18 @@
+.. TODO: change link from quick start primer to subscriber implementation page
+
+You can use the following sample application to test the code examples on this
+page. To use the sample application, perform the following steps:
+
+1. Ensure you have the {+driver-short+} installed.
+#. Create a new Java project in your IDE.
+#. Create a package called ``helpers`` within the existing ``java`` package in
+   your project.
+#. Create a new Java file in the ``helpers`` package
+   named ``SubscriberHelpers``. Implement the ``Subscriber`` implementations, which are
```

Comment:
Not related to this comment, but in response to your broad comment, the reason I started working on usage examples now was because we're waiting on tech review on every other page. Jordan suggested working on usage examples while waiting on tech review because this page is more self contained and does not rely on other pages as much as the write ops pages do.

Reply:
makes sense!